frontend/packages/noobaa-storage-plugin/src/components/resource-providers-card/resource-providers-card.tsx

```diff
@@ -62,6 +64,16 @@ const ResourceProviders: React.FC<DashboardItemProps> = ({
     RESOURCE_PROVIDERS_QUERY.UNHEALTHY_PROVIDERS_TYPES,
     'result',
   ]);
+  const resourcesLinksResponse = prometheusResults.getIn([
+    RESOURCE_PROVIDERS_QUERY.RESOURCES_LINK_QUERY,
+    'result',
+  ]);
+
+  const noobaaSystemAddress = getMetric(resourcesLinksResponse, 'system_address');
+  const noobaaSystemName = getMetric(resourcesLinksResponse, 'system_name');
+  let link = '';
+  if (!_.isNil(noobaaSystemAddress) && !_.isNil(noobaaSystemName))
```

Comment:
I think this should suffice?
```suggestion
  if (noobaaSystemAddress && noobaaSystemName)
```